test/testdrive/session.td

```diff
@@ -49,9 +49,9 @@ max_kafka_connections               1000                    "The maximum number
 max_materialized_views              100                     "The maximum number of materialized views in the region, across all schemas (Materialize)."
 max_objects_per_schema              1000                    "The maximum number of objects in a schema (Materialize)."
 max_postgres_connections            1000                    "The maximum number of PostgreSQL connections in the region, across all schemas (Materialize)."
-max_query_result_size               1073741824              "The maximum size in bytes for a single query's result (Materialize)."
+max_query_result_size               "1.0 GB"              "The maximum size in bytes for a single query's result (Materialize)."
```

Comment:
PostgreSQL doesn't put a space between the value and the unit:

```
benesch=# set log_temp_files = '10 GB';
SET
benesch=# show log_temp_files;
 log_temp_files 
----------------
 10GB
(1 row)
```

Reply:
updated

Reply:
(Looks to still have the space!)

Reply:
🤦‍♀️

Reply:
Updated for real this time!